src/jshint.js

```diff
@@ -1951,7 +1962,7 @@ var JSHINT = (function() {
           this["function"] = true;
           break;
         case "const":
-          setprop(funct, v, { unused: false });
+          block[v]["(unused)"] = false;
```

Comment:
Document the contract here of why you know `block` is not null/undefined here (to make life easier later)